setup.py

```diff
@@ -14,8 +14,6 @@
         "pytest-asyncio<0.11.0",
         "pytest-cov",
         "shapely",
-        "botocore==1.15.32",
-        "boto3==1.12.32",
```

Comment:
aioboto3, depends on aiobotocore which depends on specific boto3 and botocore versions. It's easier to let hem decide the version they need ;-)